### PR TITLE
[Work in Progress] Express server setup w/ DynamoDB

### PR DESCRIPTION
**Pull request by amanongithub7**
This is a WIP currently - do not merge this branch or commit to this remote branch

### CODE REVIEW

File: server/types/userTypes.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export interface User{
+    username: string,
+    password: string,
+    score: number,
+    questionsAnswered: number[],
+    favoriteDessert: string,
```
Comment (JensonChoi) [inline server/types/userTypes.ts:6]: why does this matter again?

File: server/routes/users.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+/* GET all users. */
+router.get('/', function(req: Request, res: Response, next: NextFunction) {
+  res.send('respond with a resource');
+});
+
+/* POST new user details on db - aka create user */
+router.post('/', async function(req: Request, res: Response, next: NextFunction){
+  console.log("inside post")
+  const username = req.body.username;
+  const password = req.body.password;
```
Comment (JensonChoi) [inline server/routes/users.ts:20]: should hash the password (e.g. with bcrypt) before storing in db

File: server/types/userTypes.ts (hunk 1 above)
Comment (TheDarkLord247) [reply server/types/userTypes.ts:6]: lmao we need something interesting on a user's profile...

File: server/routes/users.ts (hunk 2 above)
Comment (TheDarkLord247) [reply server/routes/users.ts:20]: ye

**Pull request closed without merging**